macros/utils/table_operations/insert_rows.sql

```diff
@@ -0,0 +1,75 @@
+{% macro insert_rows(table_relation, rows, should_commit=false) %}
+    {% if not table_relation %}
+        {{ elementary.edr_log('Recieved table relation is not valid (make sure elementary models were executed successfully first)') }}
+        {{ return(none) }}
+    {% endif %}
+
+    {% set columns = adapter.get_columns_in_relation(table_relation) -%}
+    {% if not columns %}
+        {% set table_name = elementary.relation_to_full_name(table_relation) %}
+        {{ elementary.edr_log('Could not extract columns for table - ' ~ table_name ~ ' (might be a permissions issue)') }}
+        {{ return(none) }}
+    {% endif %}
+
+    {% set insert_rows_queries = elementary.get_insert_rows_queries(table_relation, columns, rows) %}
+    {% for insert_query in insert_rows_queries %}
+      {% do elementary.debug_log("[%d/%d] Running insert query." % (loop.index, insert_rows_queries | length)) %}
+      {% do dbt.run_query(insert_query) %}
+    {% endfor %}
+    {%- if should_commit -%}
+        {% do adapter.commit() %}
+    {%- endif -%}
+{% endmacro %}
+
+{% macro get_insert_rows_queries(table_relation, columns, rows, max_query_size=1000000) -%}
```

Comment:
Can we make the max_query_size configurable? I'd call it insert_query_max_size or something along these lines